src/hooks/hooks.ts

```diff
@@ -18,28 +19,27 @@ AfterAll(async () => {
   await browser.close();
 });
 
-Before(async function (this: any) { // Access 'this' for scenario context
-  console.log('Creating new context and page...');
-  Context = await browser.newContext();
-  const page = await Context.newPage();
-  pageFixture.page = page; 
+Before(async function (this: any) {
```

Comment:
create an interface then refer it to this "this" instead of type any. 

```
interface TestContext {
  page: Page;
}

Before(async function (this: TestContext) {
  adminContext = await browser.newContext({
    storageState: authConfig.admin.storageState
  });
  adminPage = await adminContext.newPage();
  pageFixture.adminPage = adminPage;
  this.page = adminPage;
});
```

Reply:
Done